src/test/java/lesson6/task9/DivisibilityCheckerTest.java

```diff
@@ -0,0 +1,21 @@
+package lesson6.task9;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class DivisibilityCheckerTest {
+
+    @Test
+    void shouldCheckDivisibilityByThree() {
```

Comment:
you are testing positive and negative scenario in one test, thats a bad practice